lib/src/base-styles/_nx-color-swatches.scss

```diff
@@ -38,6 +38,7 @@
   --nx-swatch-grey-85: hsl(var(--nx-swatch-grey-hs), 85%);
   --nx-swatch-grey-90: hsl(var(--nx-swatch-grey-hs), 90%);
   --nx-swatch-grey-95: hsl(var(--nx-swatch-grey-hs), 95%);
+  --nx-swatch-grey-97: hsl(var(--nx-swatch-grey-hs), 97%);
```

Comment:
This isn't required once you get latest changes from the feature branch.

Reply:
> This isn't required once you get latest changes from the feature branch.

Thanks!